draft-tuexen-opsawg-pcapng.xml

```diff
@@ -2576,6 +2780,12 @@ Section Header
             <xref target="section_dsb">Decryption Secrets Block</xref>
           </c>
 
+
+          <c>0x0000000B/c>
```

Comment:
That needs to be

    <c>0x0000000B</c>

There's a missing `<` before the `/c>`